tests/customer-resource-show-test.js

```diff
@@ -166,19 +172,19 @@ describeApplication('CustomerResourceShow', () => {
       });
 
       return this.visit(`/eholdings/customer-resources/${resource.id}`, () => {
-        expect(ResourcePage.$root).to.exist;
+        expect(ResourcePage).to.exist;
       });
     });
 
     it('displays the title name', () => {
-      expect(ResourcePage.titleName).to.equal(resource.title.name);
+      expect(ResourcePage.titleName).to.equal('Best Title Ever');
     });
 
-    it.still('does not display a content type', () => {
-      expect(ResourcePage.contentType).to.equal('');
+    it('does not display a content type', () => {
```

Comment:
We'll want `it.always` now instead of `it.still`.

Basically, the page will be rendered without the content type element because the request is still out. So this test will pass immediately while there's a loading indicator on the page.

`it.always` will make sure that there won't be a content type for at least 100ms

Reply:
Using `it.always` now